download/src/index.js

```diff
@@ -86,7 +87,18 @@ async function download() {
       try {
         const name = platformToName[platform];
         const url = `https://github.com/zeit/now-cli/releases/download/${packageJSON.version}/${name}.gz`;
-        const resp = await fetch(url, { compress: false });
+        // check whether the proxy was set and configure the proxy agent settings
```

Comment:
The `proxy-agent` module also does the env var parsing logic. See https://github.com/TooTallNate/node-proxy-agent/pull/16.